pkgs/tools/admin/trivy/default.nix

```diff
@@ -29,9 +32,22 @@ buildGoModule rec {
     "-X=github.com/aquasecurity/trivy/pkg/version.ver=v${version}"
   ];
 
+  nativeBuildInputs = [ installShellFiles ];
+
   # Tests require network access
   doCheck = false;
 
+  postInstall =
+    let
+      trivy = if stdenvNoCC.buildPlatform.canExecute stdenvNoCC.hostPlatform then "\"$out\"" else buildPackages.trivy;
```

Comment:
```suggestion
      trivy = if stdenv.buildPlatform.canExecute stdenv.hostPlatform then placeholder "out" else buildPackages.trivy;
```

Reply:
Done. I used stdenvNoCC since I thought we don't need a C compiler here, hence use the "smaller" package. I guess that dosen't matter in the end?